tasks-by-users/AbbasShaikh0304/01-E-Commerce website/Style.css

```diff
@@ -0,0 +1,601 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+body {
+  font-family: "Poppins", sans-serif;
+}
+.navbar {
+  display: flex;
+  align-items: center;
+  padding: 20px;
+}
+nav {
+  flex: 1;
+  text-align: right;
+}
+nav ul {
+  display: inline-block;
+
+  list-style-type: none;
+}
+nav ul li {
+  display: inline-block;
+
+  margin-right: 20px;
+}
+a {
+  color: #555;
+  text-decoration: none;
+}
+p {
+  color: #555;
+}
+.container {
+  max-width: 1300px;
+
+  margin: auto;
+  padding-left: 25px;
+  padding-right: 25px;
+}
+.row {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: space-around;
+}
+.col-2 {
+  flex-basis: 50%;
+
+  min-width: 300px;
+}
+.col-2 img {
+  max-width: 100%;
+
+  padding: 50px 0;
+}
+.col-2 h1 {
+  font-size: 50px;
+  line-height: 60px;
+
+  margin: 25px 0;
+}
+.btn {
+  display: inline-block;
+
+  margin: 30px 0;
+  padding: 8px 30px;
+
+  color: #fff;
+  background: #ff523b;
+
+  border-radius: 30px;
+}
+.btn:hover {
+  background: #563434;
+
+  transition: background 0.5s;
+}
+.header {
+  background: radial-gradient(#fff, #ffd6d6);
+}
+.header .row {
+  margin-top: 70px;
+}
+.categories {
+  margin: 70px 0;
+}
+.col-3 {
+  flex-basis: 30%;
+  min-width: 250px;
+
+  margin-bottom: 30px;
+}
+.col-3 img {
+  width: 100%;
+}
+.small-container {
+  max-width: 1080px;
+
+  margin: auto;
+  padding-left: 25px;
+  padding-right: 25px;
+}
+.col-4 {
+  flex-basis: 25%;
+  min-width: 250px;
+
+  margin-bottom: 50px;
+  padding: 10px;
+
+  transition: transform 0.5s;
+
+  cursor: pointer;
+}
+.col-4:hover {
+  transform: translateY(-5px);
+}
+.col-4 img {
+  width: 100%;
+}
+.title {
+  position: relative;
+  text-align: center;
```

Comment:
here @AbbasShaikh0304

Reply:
instead of adding here and there, would you please suggest me the lines? it will be easy and helpfull for me to commit the changes. @Tauqeer-Ahmed-99